tests/conftest.py

```diff
@@ -73,6 +74,13 @@ def create_batch_db(db_engine):
     batch.Base.metadata.drop_all(db_engine)
 
 
+@pytest.fixture(scope="session")
+def create_cohort_db(db_engine):
```

Comment:
Know this is knit picky but would you be able to use [the summary/Args/Yields](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) style? Also I don't think you need as much detail about how a fixture works but rather what this one does. i.e. what is in a cohort db (maybe a summary of tables?). Def make it clear that the db will be dropped on cleanup but as a reminder this is a _session_ fixture. So it will be cleaned up when the entire session has finished running, not when the (test) function exits.

Reply:
Thanks for taking the time to speak with me about this. All the requested updates on this are done.